src/containers/User/UserArea.tsx

```diff
@@ -108,14 +198,21 @@ const OrgSelector: React.FC<{ user: User; orgs: OrganisationSimple[]; onLogin: F
 
 const UserMenu: React.FC<{ user: User }> = ({ user }) => {
   const { logout } = useUserState()
+  const { push } = useRouter()
   return (
     <div id="user-menu">
-      <Button animated onClick={() => logout()}>
+      <Button>
         <Button.Content visible>
-          {user?.firstName || ''} {user?.lastName || ''}
-        </Button.Content>
-        <Button.Content hidden>
-          <Icon name="log out" />
+          <Dropdown text={`${user?.firstName || ''} ${user?.lastName || ''}`}>
+            <Dropdown.Menu>
+              <Dropdown.Item
+                icon="edit"
+                text={strings.MENU_EDIT_USER}
+                onClick={() => push('/application/new?type=UserEdit')}
```

Comment:
Should this be included in some config?

Reply:
What do you mean?

Reply:
UserEdit - if code changes we need to find where is used and change.
Would be better to have that as a config that users can easily change in case they would like to have another one instead.